doc/install.md

```diff
@@ -39,19 +39,19 @@ GOOGLE_ACCOUNT_PASSWORD = "<a valid google password>"
 ```bash
 docker-compose up -d
 # Once exodus started, you can check its logs
-docker logs -f exodus
+docker-compose logs -f exodus
```

Comment:
This does not seem to work for me, as it just tells me 

```
ERROR: No such service: exodus
```

Same for every command `docker-compose` with the service exodus.

Shouldn't it be `exodus-worker` or `exodus-front` here?

Reply:
Have to check, it's been a while ^^